tools/roles/pytorch/vars/main.yml

```diff
@@ -47,6 +60,14 @@ omnia_foler_mode: "0755"
 
 # Errors: pytorch_install.yml
 error_pull_container: "Unable to pull container image"
+pytorch_cpu_definition_fail_msg: |
+  "Failed. pytorch_cpu details is not defined in pytorch.json. Verify the pytorch.json and re-run the playbook."
+pytorch_amd_definition_fail_msg: |
+  "Failed. AMD GPU detected and pytorch_amd details is not defined in pytorch.json. Verify the pytorch.json and re-run the playbook."
+pytorch_nvidia_definition_fail_msg: |
+  "Failed. NVIDIA GPU detected and pytorch_nvidia details is not defined in pytorch.json. Verify the pytorch.json and re-run the playbook."
+pytorch_gaudi_definition_fail_msg: |
+  "Failed. Intel GAUDI GPU detected and pytorch_gaudi details is not defined in pytorch.json. Verify the pytorch.json and re-run the playbook."
```

Comment:
Gaudi

Reply:
Addressed changes in internal PR. It will be available in next set of PRs